cmd/collector/main.go

```diff
@@ -21,12 +21,24 @@ import (
 	"os"
 	_ "time/tzdata"
 
+	"github.com/google/uuid"
 	"github.com/observiq/observiq-otel-collector/collector"
 	"github.com/observiq/observiq-otel-collector/internal/logging"
 	"github.com/observiq/observiq-otel-collector/internal/service"
 	"github.com/observiq/observiq-otel-collector/internal/version"
+	"github.com/observiq/observiq-otel-collector/opamp"
 	"github.com/spf13/pflag"
 	"go.uber.org/zap"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	// env variable name constants
+	endpointENV  = "OPAMP_ENDPOINT"
```

Comment:
nit: should this not be uppercase?

Reply:
For Go if there's an abbreviation in a variable name it should be fully capitalized. ENV could probably go either way so if we want to change it to `Env` that's fine.